.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,34 @@
+name: Feature request
+description: "新機能の提案"
+labels: ["enhancement"]
+body:
+
+  - type: markdown
+    attributes:
+      value: |
+        [Issueの立て方](https://github.com/TUS-OSK/quantum-gomoku/tree/develop/docs/0.HowToDev/02.DevProcess#issue%E3%81%AE%E7%AB%8B%E3%81%A6%E6%96%B9)を参考に記述してください。
+  - type: textarea
+    attributes:
+      label: "Problem"
+      description: "詳細を記述してください。"
+      placeholder: |
+        以下のようなことを添えて記述してください。
+        - issue number
+          - 例: fixed #123
```

新規機能開発なので、issue numを書くことはあまりないと思います
```suggestion
        - issue number（強い依存関係がある場合は記載する）
          - 例: after #123
```